lib/axiom/fs/base/execute_context.js

```diff
@@ -203,6 +200,9 @@ ExecuteContext.prototype.setCallee = function(executeContext) {
  * @return {void}
  */
 ExecuteContext.prototype.setStdio = function(stdio) {
+  if (this.stdio)
+    this.stdio.signal.onData.removeListener(this.dispatchSignal_);
```

Comment:
I think you need to add `this` as 2nd parameter, to match the `addListener` call